Runtime/SpriteSwitchTrack.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Unity.SelectionGroups;
+using UnityEngine.Playables;
+using UnityEngine.Timeline;
+
+namespace Unity.MaterialSwitch
+{
+    /// <summary>
+    /// Tracks for changing between sprites parameters. 
+    /// </summary>
+    [TrackClipType(typeof(SpriteSwitchClip))]
+    [TrackBindingType(typeof(SelectionGroup))]
+    public class SpriteSwitchTrack : TrackAsset
```

Comment:
Do we need this to be public ?

Reply:
I'm not sure. It matches the accessability of the corresponding MaterialSwitch classes.

Reply:
Please change to internal as we discussed.